client/src/featureform/list.py

```diff
@@ -67,7 +67,7 @@ def list_name_variant_status_desc(stub, resource_type):
             searchNameVariant = metadata_pb2.NameVariant(name=f.name, variant=v)
             for x in stub_list_functions[resource_type][1](iter([searchNameVariant])):
                 if x.variant == f.default_variant:
-                    format_rows(f.name, f"{f.default_variant} (default)", f.status.Status._enum_type.values[f.status.status].name, x.description)
+                    format_rows(f.name, f"{f.default_variant} (default)", x.status.Status._enum_type.values[x.status.status].name, x.description)
```

Comment:
can we use a descriptive name instead of x if it isn't too much work?

Reply:
i agree with ahmad but this file is fucked so we can worry about it later

Reply:
Just to leave a paper trail: given the need to refactor the various responsibilities in the client, we've decided on a series of steps to set the stage for this refactor. Once we earn some time to tackle this work, these single-letter variables will go the way of the dodo.